data_hub_api/kotahi_docmaps/v1/codecs/evaluation.py

```diff
@@ -67,13 +67,15 @@ def get_evaluation_and_type_list_from_email_body(
 ) -> Optional[Sequence[dict]]:
     if email_body:
         evalution_list = []
+
         evaluation_summary_text = extract_elife_assessments_from_email(email_body)
         if evaluation_summary_text:
             evaluation_summary_dict = {
                 'evaluation_type': DOCMAP_EVALUATION_TYPE_FOR_EVALUATION_SUMMARY,
                 'evaluation_text': evaluation_summary_text
             }
             evalution_list.append(evaluation_summary_dict)
+
         public_reviews = extract_elife_public_reviews_from_email(email_body)
         if public_reviews:
             public_review_parts = extract_public_review_parts(public_reviews)
```

Comment:
I think here also it is not obvious that an empty value means joint review? Could there not be other reasons for it to be empty?

Reply:
If `public_review_parts` is not defined then that means it doesn't have seperate parts so just one part then it is a `joint review`.
But if `public_reviews` is empty it will be totaly other case, there is no reviews yet. So function will return empty list.

Reply:
If public_review_parts is not defined then that means it doesn't have seperate parts so just one part then it is a joint review.
But if public_reviews is empty it will be totaly other case, there is no reviews yet. So function will return empty list.